src/app/modules/myinfo/myinfo.service.ts

```diff
@@ -451,4 +458,44 @@ export class MyInfoService {
       },
     )()
   }
+
+  /**
+   * Extracts myInfo data using the provided form and the cookies of the request
+   * @param form the form to validate
+   * @param cookies cookies of the request
+   * @returns ok(MyInfoData) if the form has been validated successfully
+   * @returns err(MyInfoMissingAccessTokenError) if no myInfoCookie was found on the request
+   * @returns err(MyInfoCookieStateError) if cookie was not successful
+   * @returns err(MyInfoNoESrvcIdError) if form has no eserviceId
+   * @returns err(MyInfoAuthTypeError) if the client was not authenticated using MyInfo
+   * @returns err(MyInfoCircuitBreakerError) if circuit breaker was active
+   * @returns err(MyInfoFetchError) if validated but the data could not be retrieved
+   * @returns err(MissingFeatureError) if using an outdated version that does not support myInfo
+   */
+  extractMyInfoData(
```

Comment:
can we rename this to start with `get` or `fetch` or `retrieve` to reflect the fact that it makes an API call to MyInfo?